FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -68,5 +67,78 @@ public void ExecuteTestInvalidCredentials()
+        {
+            var connString =
+                $"database={Database};username={Username};password={Password};endpoint={Endpoint};account={Account}";
+
+            using var conn = new FireboltConnection(connString);
+            conn.Open();
+            conn.SetEngine(Engine);
+            var command = conn.CreateCursor();
+            command.Execute("SELECT '2022-05-10 23:01:02.123455'::timestampntz");
+            DateTime dt = new DateTime(2022, 5, 10, 23, 1, 2, 0).AddTicks(1234550);
+            NewMeta newMeta = ResponseUtilities.getFirstRow(command.Response);
+            Assert.That(newMeta.Meta, Is.EqualTo("TimestampNtz"));
+            Assert.That(newMeta.Data[0], Is.EqualTo(dt));
+        }
+
+        [Test]
+        public void ExecuteSelectTimestampPgDate()
+        {
+            var connString =
+                $"database={Database};username={Username};password={Password};endpoint={Endpoint};account={Account}";
+
+            using var conn = new FireboltConnection(connString);
+            conn.Open();
+            conn.SetEngine(Engine);
+            var command = conn.CreateCursor();
+            command.Execute("SELECT '2022-05-10'::pgdate");
+            DateTime dt = new DateTime(2022, 5, 10, 0, 0, 0);
+            NewMeta newMeta = ResponseUtilities.getFirstRow(command.Response);
+            Assert.That(newMeta.Meta, Is.EqualTo("Date"));
+            Assert.That(newMeta.Data[0], Is.EqualTo(DateOnly.FromDateTime(dt)));
+        }
+
+        [Test]
+        public void ExecuteSelectTimestampTz()
+        {
+            var connString =
+                $"database={Database};username={Username};password={Password};endpoint={Endpoint};account={Account}";
+
+            using var conn = new FireboltConnection(connString);
+            conn.Open();
+            conn.SetEngine(Engine);
+            var command = conn.CreateCursor();
+            command.Execute("SELECT '2022-05-10 23:01:02.123456 Europe/Berlin'::timestamptz");
+
+            DateTime dt = DateTime.Parse("2022-05-10 21:01:02.123456Z");
+            NewMeta newMeta = ResponseUtilities.getFirstRow(command.Response);
+            Assert.That(newMeta.Data[0], Is.EqualTo(dt));
+            Assert.That(newMeta.Meta, Is.EqualTo("TimestampTz"));
+
+        }
+
+        [Test]
+        public void ExecuteSelectTimestampTzWithDifferentFormatTz()
+        {
+            var connString =
+                $"database={Database};username={Username};password={Password};endpoint={Endpoint};account={Account}";
+            using var conn = new FireboltConnection(connString);
+            conn.Open();
+            conn.SetEngine(Engine);
+            var command = conn.CreateCursor();
+            command.Execute("SET advanced_mode=1");
+            command.Execute("SET time_zone=Asia/Calcutta");
+            command.Execute("SELECT '2022-05-11 23:01:02.123123 Europe/Berlin'::timestamptz");
+
+            DateTime dt = DateTime.Parse("2022-05-11 21:01:02.123123Z");
```

Comment:
Is this correct? `Asia/Calcutta` 's offset is 5:30 from GMT. 
![image](https://user-images.githubusercontent.com/93913847/212641934-7aaa4e75-f60e-4cc5-a85b-6dde8ae15bc9.png)

Reply:
When you do SELECT '2022-05-11 23:01:02.123123 Europe/Berlin', you actually get the server response in the Asia/calcutta timezone.
The response from the server is: "2022-05-12 02:31:02.123123+05:30", which in zulu time is as below
![image](https://user-images.githubusercontent.com/10419887/212658302-1d14d3ce-43aa-4ffb-95e8-1ed4d8cc6b94.png)

I also gave it a go with Postgres and I'm getting the same result (it's just showing it for my current tz):
![image](https://user-images.githubusercontent.com/10419887/212657885-1a10e546-4f52-4e4d-aa6f-c18d984073e0.png)